code/modules/surgery/organs_internal.dm

```diff
@@ -58,13 +61,12 @@
 	var/obj/item/organ/external/affected = target.get_organ(target_zone)
 	for(var/obj/item/organ/internal/I in affected.internal_organs)
 		if(I && I.damage > 0 && !BP_IS_PROSTHETIC(I) && (I.surface_accessible || affected.how_open() >= (affected.encased ? SURGERY_ENCASED : SURGERY_RETRACTED)))
-			if(I.status & ORGAN_DEAD && I.can_recover())
-				user.visible_message("<span class='notice'>\The [user] treats damage to [target]'s [I.name] with [tool_name], though it needs to be recovered further.</span>", \
-				"<span class='notice'>You treat damage to [target]'s [I.name] with [tool_name], though it needs to be recovered further.</span>" )
+			if(I.status & ORGAN_DEAD)
+				to_chat(user, SPAN_NOTICE("You were unable to treat \the [I.name] due to its necrotic state."))
```

Comment:
`\the [I]`

Reply:
[I]'ve gone ahead and removed the rest as well